pkgs/by-name/ro/ropebwt2/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zlib,
+}:
+
+stdenv.mkDerivation {
+  name = "ropebwt2";
+  version = "0-unstable-2021-02-01";
+
+  src = fetchFromGitHub {
+    owner = "lh3";
+    repo = "ropebwt2";
+    rev = "bd8dbd3db2e9e3cff74acc2907c0742c9ebbf033";
+    hash = "sha256-R/VvbprwcfXF2TBZOYmc1MU3AzCcXFfWCHoYYumXtI8=";
+  };
+
+  buildInputs = [ zlib ];
+  installPhase = ''
```

Comment:
```suggestion
  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
  installPhase = ''
```

Reply:
Okay, I've added the changes, thanks a lot !